js/lib/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+sha="a3616d74c0b9b9409d580337e04c32da6e3877e4" # v2.5.1
+mkdir -p ./corejs-repo
+cd ./corejs-repo
+git init
+git remote add origin https://github.com/zloirock/core-js.git
+git fetch origin --depth=1 "${sha}"
+git reset --hard "${sha}"
+cp ../k6-shim.js shim.js
+cp ../corejs-build.sh corejs-build.sh
+docker run --user "$UID:$GID" --rm -ti  -v $(pwd):/opt/g --entrypoint /opt/g/corejs-build.sh --workdir /opt/g node
```

Comment:
It would be safer to quote, and also use `id` in case the env vars aren't defined:

```suggestion
docker run --user "$(id -u):$(id -g)" --rm -ti  -v "$(pwd):/opt/g" --entrypoint /opt/g/corejs-build.sh --workdir /opt/g node
```

But since we're using Docker for this, could we do everything there, including the `git` and `rice` commands? I don't have `rice` installed locally, so it would be good if all of this is contained. :)

I would also suggest using Bash and `set -euo pipefail`, but I think you're avoiding that to do the cleanup, which is fine, but you could also do it in Bash with `trap`. Not a big deal though if you don't want to complicate it.

Reply:
I prefer not to complicate it ... predominantly because I don't think anyone will run this again more than once or twice, so spending a lot of time making it awesome seems unnecessary when in reality whoever runs it (probably to drop even more stuff in the future) will run it once and then we will either never touch it again or completely drop it. 

This very much depends on what the discussion about https://github.com/loadimpact/k6/pull/1772#issuecomment-745417009 revolves around ... if we drop the metadata stuff, the rest is standard and likely fairly easy to implement, so I will vote for taking a week to implement parts of it and get them in goja and then repeat until all of them are in goja and then just drop the whole core-js polyfill :tada: . But I also expect that even if this happens it won't be in the next release or soon at all